cmd/repo.go

```diff
@@ -415,11 +415,12 @@ func (r *RepositoryFile) GetDefaultRepoName(rootConfig *RootCommandConfig) (stri
 	return repoName, nil
 }
 
-func (r *RepositoryFile) Remove(name string) {
+func (r *RepositoryFile) Remove(name string, rootConfig *RootCommandConfig) {
```

Comment:
If the only thing `rootConfig` is doing is related to logging, instead of passing `RootCommandConfig`, you should pass a `LoggingConfig` :)